quickcheck-dynamic/CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ changes.
 
 ## UNRELEASED
 
+* Breaking removed `Realized`
```

Comment:
If we only deprecate `Realized` I think that would be a non-breaking change which is somewhat nicer

Reply:
I know it's probably a bit of a grey area since `quickcheck-dynamic` follows SemVer, but Haskell PVP prescribes that deprecations are breaking changes because it would cause downstream builds to fail with `-Werror`